README.md

```diff
@@ -30,14 +30,17 @@ This currently includes all cru.org styling from cru.scss.
 
 #### Branded checkout basic example
 
-Add the following code to your page where appropriate:
+Add the following code to your page where appropriate. You must change the value of the `code` and `tsys-env` attributes:
+- `code` must be changed to the designation number you want users to give to.
+- `tsys-env` must be changed to match your environment. It is provided by DPS when adding your domain to the TSYS whitelist.
 ```html
 <link rel="stylesheet" href="https://give-static.cru.org/branded-checkout.min.css">
 <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.3/css/font-awesome.min.css">
 
 <branded-checkout
     ng-app="brandedCheckout"
-    code="0763355">
+    code="0763355"
```

Comment:
out of curiosity, why'd you name this 'code' instead of 'designationNumber'? Maybe we can change this?

Reply:
Probably because other frontend code or EP called it that. Now's the time to change it before it starts getting implemented.

Reply:
EP might probably does call it that. But EP's terminology isn't as important as Cru's terminology, which is going to be more stable long-term. (There's no guarantee we'll stick with EP for more than a few years.) Think of EP as an implementation detail.

So yeah, let's change this now. Thanks!